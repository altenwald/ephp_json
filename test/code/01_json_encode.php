Empty array output as array: <?=json_encode('tada')?>