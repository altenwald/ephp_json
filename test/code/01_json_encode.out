Empty array output as array: "tada"